tests/unit/components/sl-date-time-test.js

```diff
@@ -16,15 +16,28 @@ test( 'Default class names are present', function( assert ) {
 
 test( 'Attribute "datetime" is properly set', function( assert ) {
     this.subject({
-        timezone: 'America/Chicago',
-        value: new Date( 2015, 0, 1 )
+        format: 'datetime',
+        timezone: 'America/Denver',
```

Comment:
I was specifically trying to get the test to fail. Apparently the github server where travis does the testing is in EDT (currently), so that's why the America/Chicago failed, but not locally. America/Denver isn't CDT or EDT, so it failing becomes more likely. Timezones make my head hurt.